src/EnergyPlus/FuelCellElectricGenerator.cc

```diff
@@ -3194,7 +3195,7 @@ namespace FuelCellElectricGenerator {
             this->AirSup.TairIntoBlower = 0.0;
             this->AirSup.QskinLoss = 0.0;
             this->AirSup.QintakeRecovery = 0.0;
-            this->FCPM.NumCycles = 0;
+            this->FCPM.NumCycles = this->FCPM.NumCyclesAtStart;
```

Comment:
Fix #8514 - Generator:FuelCell:PowerModule: Number of Stops at Start of Simulation is ignored